component/doc/README.md

```diff
@@ -0,0 +1,346 @@
+#Configuring LinkedIn Authenticator
+
+This page provides instructions on how to configure the LinkedIn 
+authenticator and the WSO2 Identity Server using a sample app to 
+demonstrate authentication.You can find more information in the 
+following sections.
+
+> This is tested for the LinkedIn API version 2.0. LinkedIn 
+Authenticator is supported by Identity Server 5.9.0 upwards.
```

Comment:
This authenticator has been tested with LinkedIn API version 2.0. The LinkedIn Authenticator is supported from WSO2 Identity Server 5.9.0 onwards.